nodejs/packages/layer/package.json

```diff
@@ -26,28 +26,28 @@
     "node": ">=10.0.0"
   },
   "dependencies": {
-    "@opentelemetry/api": "^1.0.0-rc.0",
-    "@opentelemetry/exporter-collector-proto": "^0.18.2",
-    "@opentelemetry/instrumentation": "^0.18.2",
+    "@opentelemetry/api": "1.0.0-rc.0",
+    "@opentelemetry/exporter-collector-proto": "^0.19.0",
+    "@opentelemetry/instrumentation": "^0.19.0",
     "@opentelemetry/instrumentation-aws-lambda": "https://gitpkg.now.sh/anuraaga/opentelemetry-js-contrib/plugins/node/opentelemetry-instrumentation-aws-lambda?2f43915dd94f21a4daf54632db2c12c1ca597a6d",
```

Comment:
Unfortunately the parenting from Lambda has been merged but not released yet. I have backported the change to 0.16.0 here, can you try it?

https://gitpkg.now.sh/anuraaga/opentelemetry-js-contrib/plugins/node/opentelemetry-instrumentation-aws-lambda?515bfc745f243115ea1820d20f137438d606a5eb

https://github.com/anuraaga/opentelemetry-js-contrib/tree/aws-lambda-0.16